System/Posix/Terminal/Common.hsc

```diff
@@ -48,7 +49,100 @@ module System.Posix.Terminal.Common (
   minInput,
   withMinInput,
 
-  BaudRate(..),
+  BaudRate,
```

Comment:
I'd like to see this export list maintained in just one module.  Perhaps `System.Postfix.Terminal.Exports` with `Common` and `ByteString` both re-exporting `Exports` or something along these lines...

Reply:
Seems fine to me, but what's the rationale?